CODE_OF_CONDUCT.md

```diff
@@ -14,22 +14,22 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
 include:
 
-* Using welcoming and inclusive language
```

Comment:
Same for this, no need to format, or?

Reply:
See discussion above.

Reply:
Somehow I cannot mark this thread as resolved, but please consider it to be resolved.